addons/OPT/isDev.hpp

```diff
@@ -0,0 +1,5 @@
+// This file will be used by the build script in order to set or unset the ISDEV macro flag
+// Therefore the content in this file is not relevant when using the build script (the scripts will overwrite it)
+
+// disable dev-mode by default, in case someone builds manually
+#undef ISDEV
```

Comment:
ISDEV ist glaube ich nicht ausreichend, wir bräuchten für ein sinnvolles Debugging auch DEBUGFULL; da nur mit FULLDEV auch Funktionsaufrufe mit Übergabewerten geloggt werden. Das würde ich auf jeden Fall wollen, ISDEV gibt nur Warnungen, wenn eine Funktion nicht existiert, glaube ich jedenfalls noch aus dem Gedächtnis zu wissen

Reply:
Glaube mir `DEBUGFULL` willst du nicht standardmäßig aktiviert haben. Dann findest du im Log nämlich gar nichts mehr.
`ISDEV` produziert schon echt dicke Log-Dateien mit verdammt vielen Infos, aber `DEBUGFULL` schreibt dir den kompletten Stacktrace jedes einzelnen Schritts in die Log-Datei. Das ist nützlich, wenn man ganz konkret debuggen will, aber sonst taugt das nicht.

Deswegen würde ich standardmäßig ohne `DEBUGFULL` arbeiten und wer es zum Testen braucht, kann ja die entsprechende Zeile in der `macros.hpp` entkommentieren und dann neu bauen...

Reply:
Ok, dann schauen wir uns das so an